### ISSUE TITLE
(🐞) Empty reveal locals should have a better output

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#12388]
```py
reveal_locals()
```
```
> mypy test.py
test.py:1:1: note: Revealed local types are:
Success: no issues found in 1 source file
```

This looks like a bug, I would expect something like:
```
> mypy test.py
test.py:1:1: note: There are no locals to reveal
Success: no issues found in 1 source file
```
